com.unity.render-pipelines.universal/Shaders/LitInput.hlsl

```diff
@@ -17,7 +17,7 @@ half _BumpScale;
 half _OcclusionStrength;
 #if defined(_CLEARCOAT) || defined(_CLEARCOATMAP)
 half _ClearCoatMask;
-//half _ClearCoatSmoothness; // TODO: enable
+half _ClearCoatSmoothness;
 #endif
```

Comment:
be aware: for SRP batcher, the UnityPerMaterial cbuffer must be the same for all shader passes. If we ifdef the material cbuffer and we have different keywords it might be non-intuitive to find why SRP batcher is not compatible.

Reply:
Good point. I removed the ifdfef.